components/build/jvm-build-service/operator-images.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: hacbs-jvm-operator
+  namespace: jvm-build-service
```

Comment:
Remove this line, it's probably still trying to apply on original namespace, namespace is not required.